packages/server/src/adapters/fetch/fetchRequestHandler.ts

```diff
@@ -19,7 +21,7 @@ export async function fetchRequestHandler<TRouter extends AnyRouter>(
   };
 
   const url = new URL(opts.req.url);
-  const path = url.pathname.slice(opts.endpoint.length + 1);
+  const path = opts.path ?? url.pathname.slice(opts.endpoint.length + 1);
```

Comment:
This was to enable using the `$trpc.ts` filename in remix name similar to how next has `[trpc].ts` files. This seemed like the easiest solution since finding the path is easier than knowing where the endpoint is automatically